jonprl-mode.el

```diff
@@ -48,6 +48,23 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
+   ("fix" . ?μ)
```

Comment:
Hmm, that's a good point. Do you mean this would be a separate facility not using `fix` directly? One possibility would be to use a fancy looking `Y` or maybe `⊔` with the intuition being `μ f  ≃ ⊔ (f^ω ⊥)`.